package.json

```diff
@@ -34,7 +34,7 @@
     "ensure-uploads-dir-exists": "node ./bin/ensure-uploads-dir-exists.js",
     "fontello": "gulp fontello",
     "install-dependencies": "node ./bin/install-deps.js",
-    "lint:watch": "esw --cache --watch .",
+    "lint:watch": "esw --cache --watch --quiet .",
```

Comment:
The new version now supports `--quiet` which gets rid of repeated "all good" messages.